Week2/homework/map-filter.js

```diff
@@ -1,8 +1,9 @@
 'use strict';
 
 function doubleOddNumbers(numbers) {
-  // Replace this comment and the next line with your code
-  console.log(numbers);
+  const newNumbers = numbers.filter(item => item % 2 !== 0);
+  const newNumbersAfterFilter = newNumbers.map(item => item * 2);
```

Comment:
you can use more descriptive variable names here, such as `oddNumbers = ...` and `doubledNumbers = ...`.